WalletWasabi.Tests/UnitTests/Tor/Control/TorControlReplyReaderTests.cs

```diff
@@ -60,7 +60,7 @@ public async Task InvalidStatusCodeAsync()
 		[InlineData("Unknown status code: '2 O'.", "2 OK\r\n")]
 		public async Task InvalidRepliesAsync(string expectedExceptionMsg, string data)
 		{
-			var ex = await Assert.ThrowsAsync<TorControlReplyParseException>(async () => await ParseAsync(data).ConfigureAwait(false));
```

Comment:
CAF(false) is in an anonymous function `async () => await ParseAsync(data).ConfigureAwait(false)` so the CAF(false) is correct, as it is not used on the top level